lib/elixir/lib/calendar/datetime.ex

```diff
@@ -665,6 +665,8 @@ defmodule DateTime do
   Subtracts `datetime2` from `datetime1`.
 
   The answer can be returned in any `unit` available from `t:System.time_unit/0`.
+  
+  Leaps in time are not considered, such as leap seconds or leap days.
```

Comment:
Are you sure about leap days? The `to_iso_days` conversion takes leap days into account.

Reply:
Only leap seconds are not taken into account. leap days always and everywhere.